tools/packages/bulk-test/src/findBulkTests.ts

```diff
@@ -57,16 +57,12 @@ async function findGlobFiles(
   exclude: string[] | undefined,
 ): Promise<string[]> {
   const fullBaseDir = path.resolve(baseDir);
-  const cwd = process.cwd();
   const skip = exclude ?? [];
-  try {
-    process.chdir(fullBaseDir);
-    const futurePaths =
-      globs?.map(g => glob(g, { ignore: ["node_modules/**"] })) ?? [];
-    const pathSets = await Promise.all(futurePaths);
-    const filePaths = pathSets.flat();
-    return filePaths.filter(p => !skip.some(s => p.includes(s)));
-  } finally {
-    process.chdir(cwd);
-  }
+  const futurePaths =
+    globs?.map(g =>
+      glob(g, { ignore: ["node_modules/**"], cwd: fullBaseDir }),
+    ) ?? [];
```

Comment:
Yay, no need to use `.chdir`! The `glob` function can do that for us.

On that note: Beware of the glob function returning relative paths. One needs to distinguish between "relative to the wesl.toml directory" and "relative to the wesl root".